functions/New-DbaDatabase.ps1

```diff
@@ -0,0 +1,469 @@
+function New-DbaDatabase 
+{
+  <#
+.SYNOPSIS 
+New-DbaADatabase creates a new database
+
+.DESCRIPTION
+New-DbaDatabase creates a new database with a single user filegroup, and the PRIMARY filegroup reserved for system objects.
+It allows creation with multiple files, and sets all growth settings to be fixed size rather than percentage growth.
+
+.PARAMETER SqlInstance
+The SQL server instances to be connected to; can be passed in via the pipeline.
+
+.PARAMETER PsCredential
+Allows you to login to servers using SQL Logins as opposed to Windows Auth/Integrated/Trusted. To use:
+$scred = Get-Credential, then pass $scred object to the -SqlCredential parameter. 
+To connect as a different Windows user, run PowerShell as that user.
+
+.PARAMETER DatabaseName
+The name of the new database to be created.
+
+.PARAMETER NumberOfFilesInUserFilegroup
+The number of files to create in the user filegroup for the database.  Default is 1.
+
+.PARAMETER UseDefaultFileLocations
```

Comment:
Isn't this parameter redundant? 
If no DataFileLocation or LogFileLocation is given, default has to be used.